packages/web-app-files/src/components/SideBar/Shares/Collaborators/EditDropdown.vue

```diff
@@ -4,6 +4,7 @@
       <oc-button
         :id="editShareBtnId"
         class="collaborator-edit-dropdown-options-btn"
+        :aria-label="isLocked ? dropButtonTooltip : $gettext('Open context menu to edit share')"
```

Comment:
```suggestion
        :aria-label="isLocked ? dropButtonTooltip : $gettext('Open context menu with share editing options')"
```
not so sure if this is an improvement over your text... but I'm also not happy yet with the wording `to edit share` because opening the context menu only shows options, doesn't trigger any editing.